### PR TITLE
white background

### PR DESCRIPTION
**Pull request by db0sch**
white background. it's cleaner.

@paulgaumer can you give me your feedback?

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,6 +1,6 @@
 body {
   height: 100vh;
-  background-color: red;
+  background-color: white;
```
Comment (paulgaumer) [inline style.css]: Please change title color to black

File: style.css [hunk 2]
```diff
@@ -13,6 +13,7 @@ body {
 
 h1 {
   font-family: Helvetica;
+  color: white;
```
Comment (db0sch) [inline style.css]: I should put it in black

**Comment by db0sch** [comment]
Done it!
Can you merge it @paulgaumer ?

**Comment by paulgaumer** [comment]
Looks good to me!

**Pull request merged**